src/DssSpell.sol

```diff
@@ -380,9 +80,33 @@ contract DssSpell {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/e59032178a702181d82f7a2be12bd95714ff53e0/governance/votes/Community%20Executive%20vote%20-%20January%2022%2C%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/3e18ba4514db068a96ea277e006028831dbd5ed7/governance/votes/Executive%20vote%20-%20January%2025%2C%202021.md -q -O - 2>/dev/null)"
     string constant public description =
-        "2021-01-22 MakerDAO Executive Spell | Hash: 0x2d13137cf10cdd9dcf9e5047d1894608d1d3893a5c07a1c8955f0d11726b12b2";
+        "2021-01-25 MakerDAO Executive Spell | Hash: 0x6d023ee5874db51d0cdb0491d5ae3095c89a578133c44769c836efcb06f02eb0";
+
+    // MIP29: Peg Stability Module
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4fc8d21ce2122c637f3302c63d2892b572cb5c94/MIP29/mip29.md -q -O - 2>/dev/null)"
+    string constant public MIP29 = "0xfcca2f3e493a998bf7e5532cda126d106173d2ab41d6965baa08e66a2c6cd96a";
+
+    // MIP30: Farmable cUSDC Adaptor (CropJoin)
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4fc8d21ce2122c637f3302c63d2892b572cb5c94/MIP30/mip30.md -q -O - 2>/dev/null)"
+    string constant public MIP30 = "0xbdd7787c5d43e146c9ffa021ed59f9dbc01867c16216ff4a3c4341a530c18172";
+
+    // MIP13c3-SP7: Declaration of Intent - Governance Communications Domain
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4942ee3a65f4205303146388d5af48f69f5e6898/MIP13/MIP13c3-Subproposals/MIP13c3-SP7.md -q -O - 2>/dev/null)"
+    string constant public MIP13c3SP7 = "0x821e0bb3299445e27a12fa29ce5820e5cc0b633650f7d7064aeb0de6307a807e";
+
+    // MIP7c3-SP5: Onboarding Sébastien Derivaux to the Risk Domain
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4942ee3a65f4205303146388d5af48f69f5e6898/MIP7/MIP7c3-Subproposals/MIP7c3-SP5.md -q -O - 2>/dev/null)"
+    string constant public MIP7c3SP5 = "0x8dd22faf4a65225699ab45df003a363e39636954ca5f07b7300a03e68ae96d07";
+
+    // MIP7c3-SP5: Onboarding Sam MacPherson to the Smart Contracts Domain
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4fc8d21ce2122c637f3302c63d2892b572cb5c94/MIP7/MIP7c3-Subproposals/MIP7c3-SP6.md -q -O - 2>/dev/null)"
+    string constant public MIP7c3SP6 = "0x20f8c70ec4f91fa36acb3708aaf018f9bcd42529802fefbe3633d5f80a3085d8";
+
+    // MIP28c7-SP2: Onboarding @JuanJuan to the Operational Support Domain
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/4942ee3a65f4205303146388d5af48f69f5e6898/MIP28/MIP28c7-Subproposals/MIP28c7-SP2.md -q -O - 2>/dev/null)"
+    string constant public MIP28c7SP2 = "0xb2d42ad131edacd08310a05dc008447451a4aef57bd3bec87ee0a06d055174ee";
```

Comment:
Using https://emn178.github.io/online-tools/keccak_256.html it doesn't give the same hash. There is probably an extra or missing space at the end. I've removed the check from the checklist until is fixed.

Reply:
![Screenshot from 2021-01-25 12-23-33](https://user-images.githubusercontent.com/2374718/105741644-90e16600-5f08-11eb-8bd8-23ebeee91099.png)

Looks like the markdown has `&nbsp` spaces instead of regular spaces. The markdown may have been copy/pasted from the web.